src/plugins/foxPage/components/Layout.stories.tsx

```diff
@@ -23,6 +25,54 @@ const mockAsset = {
     'FOX is an ERC-20 token created by ShapeShift which serves as the governance token for the ShapeShift DAO, token holders can vote on proposals relating to the operation...',
 }
 
+const mockOtherOpportunities: FoxOpportunities = {
```

Comment:
Actually, I would already consider it the actual data, not "mocked data": the way we're going to wire this up is effectively just fetch the APY with a service as a separate field (like a GraphQL field resolver), but apart from that, the data is going to be the same.

This should probably be extracted from stories and used directly by the `<OtherOpportunities />` component